.github/workflows/ci-weekly.yml

```diff
@@ -120,24 +120,16 @@ jobs:
         yarn test-firefox:pw 
         yarn test-webkit:pw
 
+    - name: Test Coverage
+      run: coverage
```

Comment:
Does 'coverage' run all of our packages?

Reply:
Yes. It does now